internal/server/data/deviceflowauthrequest_test.go

```diff
@@ -0,0 +1,27 @@
+package data
+
+import (
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
+
+	"github.com/infrahq/infra/internal/server/models"
+)
+
+func TestDeleteExpiredDeviceFlowAuthRequests(t *testing.T) {
+	tx := setupDB(t)
+	dfar := &models.DeviceFlowAuthRequest{
+		UserCode:   "BCDFGHJK",
+		DeviceCode: "abcdefghijklmnopqrstuvwxyz123456789000",
+		ExpiresAt:  time.Now().Add(-1),
+	}
+	err := CreateDeviceFlowAuthRequest(tx, dfar)
```

Comment:
Here as well, it would be great to include non-expired rows, to show they are not deleted.